crates/core/tedge_agent/Cargo.toml

```
@@ -17,12 +17,6 @@ assets = [
     ["target/release/tedge_agent", "/usr/bin/tedge_agent", "755"],
 ]
 
-[package.metadata.deb.systemd-units]
-unit-scripts = "../../../configuration/init/systemd"
-enable = false
-start = false
-stop-on-upgrade = false
-
```

I would add a comment to tell not to use `cargo-deb` and why.

```suggestion
// Contrary to the other daemons,
// The `agent` does not use `cargo-deb`: for OTA self-update one needs a fine-control over the deb options
```

---

It still uses `cargo-deb`, however, we don't use `systemd-utils` of them. I changed the text a bit.